resources/js/components/FormField.vue

```diff
@@ -83,6 +84,7 @@ export default {
   },
 
   methods: {
+    
```

Comment:
PR-is pigem ei ole welcome stiilimuudatused, unnecessary tühi rida.

Reply:
Git merge-iga tekkisid vead.